docs/deployment/deploy-on-aws-browsers.md

```diff
@@ -0,0 +1,123 @@
+zip -r dependencies.zip ./node_modules
+```
+
+We will now upload the `dependencies.zip` as a Lambda Layer to AWS. Unfortunately, we cannot do this directly - there is a 50MB limit on direct uploads (and the compressed Chromium build is around that size itself). Instead, we'll upload it as an object into an S3 storage and provide the link to that object during the layer creation.
+
+## Updating the code
+
+We also have to slightly update the Crawlee code:
+
+- First, we pass a new `Configuration` instance to the Crawler. This way, every crawler instance we create will have its own storage and won’t interfere with other crawler instances running in your Lambda environment.
+
+```javascript title="src/main.js"
+// For more information, see https://crawlee.dev/
+import { Configuration, PlaywrightCrawler } from 'crawlee';
+import { router } from './routes.js';
+
+const startUrls = ['https://crawlee.dev'];
+
+const crawler = new PlaywrightCrawler({
+    requestHandler: router,
+// highlight-start
+}, new Configuration({
+    persistStorage: false,
+}));
+// highlight-end
+
+await crawler.run(startUrls);
+```
+
+- Now, we actually have to supply the code with the Chromium path from the `@sparticuz/chromium` package. AWS Lambda execution also lacks some hardware support for GPU acceleration etc. - you can tell Chrome about this by passing the `aws_chromium.args` to the `args` parameter.
+
+```javascript title="src/main.js"
+// For more information, see https://crawlee.dev/
+import { Configuration, PlaywrightCrawler } from 'crawlee';
+import { router } from './routes.js';
+// highlight-next-line
+import aws_chromium from '@sparticuz/chromium';
+
+const startUrls = ['https://crawlee.dev'];
+
+const crawler = new PlaywrightCrawler({
+    requestHandler: router,
+    // highlight-start
+    launchContext: {
+        launchOptions: {
+             executablePath: await aws_chromium.executablePath(),
+             args: aws_chromium.args,
+             headless: true
+        }
+    }
+    // highlight-end
+}, new Configuration({
+    persistStorage: false,
+}));
+
+```
+
+- Last but not least, we have to wrap the code in the exported `handler` function - this will become the Lambda AWS will be executing.
+
+```javascript title="src/main.js"
+import { Configuration, PlaywrightCrawler } from 'crawlee';
+import { router } from './routes.js';
+import aws_chromium from '@sparticuz/chromium';
+
+const startUrls = ['https://crawlee.dev'];
+
+// highlight-next-line
+export const handler = async (event, context) => {
+    const crawler = new PlaywrightCrawler({
+        requestHandler: router,
+        launchContext: {
+            launchOptions: {
+                executablePath: await aws_chromium.executablePath(),
+                args: aws_chromium.args,
+                headless: true
+            }
+        }
+    }, new Configuration({
+        persistStorage: false,
+    }));
+
+    await crawler.run(startUrls);
+
+    // highlight-start
```

Comment:
Duplicated highlight start 👀

Reply:
wait... maybe I don't understand how those work then? 😅 
above the handler it's just `highlight-next-line`, I don't think it opens a new block